apps/devhub/templates/devhub/includes/macros.html

```diff
@@ -76,3 +76,11 @@
     {{ form.categories.errors }}
   </div>
 {% endmacro %}
+
+{% macro url_if_listed(addon_or_version, text) %}
```

Comment:
I'd prefer to name this "link_if_listed", to indicate that it's going to return the `<a>` and not just the url.